resources/references/core-reference/dal-reference/fields-reference/index.md

```diff
@@ -0,0 +1,60 @@
+---
+nav:
+  title: Fields Reference
+  position: 10
+
+---
+
+# Fields Reference
+
+| Name                         | Description                              |
+|:-----------------------------|:-----------------------------------------|
+| BlacklistRuleField           | Stores a "BlacklistRule" value           |
```

Comment:
this file looks somehow generated. the descriptions looks like that at least :sweat_smile: maybe @Isengo1989 can shed some light here. 

what I want to say with that is, that we need to ensure, that the linked pages stay

Reply:
Sadly not, I think Dominic generated it at some point. I will see how we can do so. Will get back here soon.

Reply:
I updated the table;

-  removed deprecated fields
-   added 2 new columns 

For future reference / generation I added a GIST -> https://gist.github.com/Isengo1989/d77209134b7c178d3ba6e75386b128ee